### PR TITLE
Product rebranding

### PR DESCRIPTION
**Pull request by Rachael-E**
This PR replaces stand alone references to ArcGIS Runtime to the new product name of ArcGIS Maps SDK for Java, or equivalent for the SDKs as a whole product (ArcGIS Maps SDKs for Native Apps). See [rebranding announcement](https://www.esri.com/arcgis-blog/products/developers/announcements/introducing-the-arcgis-maps-sdks/?rsource=https%3A%2F%2Fesriurl.com%2Fmaps-sdks-announcement). 

It also replaces references to arcgisruntime organisation with the more familiar arcgis.com organisation, the url behaves the same. 

Some references are still to be updated once the new urls are live, e.g. for samples, toolkit and demos repo, along with community pages.

@jenmerritt could you have a look please?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,12 +9,12 @@
 
 [![Link: Esri Community](https://img.shields.io/badge/Esri%20Community%20Questions-2d2670?style=flat-square)](https://community.esri.com/t5/arcgis-runtime-sdk-for-java-questions/bd-p/arcgis-runtime-sdk-for-java-questions)
 ![ArcGIS Developers Twitter](https://img.shields.io/twitter/url?label=ArcGIS%20Developers&logoColor=2d2670&url=https%3A%2F%2Ftwitter.com%2FArcGISDevs)
-[![Link: ArcGIS Runtime SDKs Blog](https://img.shields.io/badge/ArcGIS%20Runtime%20SDKS%20Blog-2d2670?style=flat-square)](https://community.esri.com/t5/arcgis-runtime-sdk-for-java-questions/bd-p/arcgis-runtime-sdk-for-java-questions)
-[![Link: ArcGIS Blog for Developers](https://img.shields.io/badge/ArcGIS%20Blog%20for%20Developers-2d2670?style=flat-square)](https://community.esri.com/t5/arcgis-runtime-sdk-for-java-questions/bd-p/arcgis-runtime-sdk-for-java-questions)
+[![Link: ArcGIS Runtime SDKs Blog](https://img.shields.io/badge/ArcGIS%20Runtime%20SDKS%20Blog-2d2670?style=flat-square)](https://community.esri.com/t5/arcgis-runtime-sdks-blog/bg-p/arcgis-runtime-sdks-blog)
```
Comment (jenmerritt) [inline README.md]:
```suggestion
[![Link: ArcGIS Maps SDKs for Native Apps Blog](https://img.shields.io/badge/ArcGIS%20Runtime%20SDKS%20Blog-2d2670?style=flat-square)](https://community.esri.com/t5/arcgis-runtime-sdks-blog/bg-p/arcgis-runtime-sdks-blog)
```
We could update this for now - I imagine redirects will be going in for community as well as the repos. Tidies the wording in the meantime?

**Review by jenmerritt — APPROVED** [review]
Nice @Rachael-E , very tidy job! There was just 1 suggestion - I had the same URL in the toolkit readme and updated the link label, let me know what you think.

File: README.md (hunk 1 above)
Comment (Rachael-E) [reply README.md]: oh yeh good idea!

**Review by mbcoder — APPROVED** [review]
Not as bad as I thought it would be.  Looks good

**Pull request merged**